pkg/new-ui/v1beta1/frontend/src/styles.scss

```diff
@@ -0,0 +1,44 @@
+/* You can add global styles to this file, and also import other style files */
+@import '~kubeflow/styles.scss';
+
+body {
+  background-color: white;
+  margin: 0;
+}
+
+.experiment-title {
+  font-weight: 400;
+  font-size: 20px;
+}
+
+.experiment-subTitle {
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.experiment-bold-text {
+  font-weight: 500;
+}
+
+.experiment-container {
+  padding-top: 1.5rem;
+  margin: auto;
+}
+
+.experiment-divider {
+  margin-top: 16px;
+  margin-bottom: 32px;
+}
+
+.experiment-add-button.mat-button {
+  border: 1px solid;
+  padding-left: 10px;
+}
+
+.two-inputs {
```

Comment:
Hmm, since this class is already needed in three different places I'd rather have it in the global styles.scss and remove it from the `trial-template.component.scss` file instead

Reply:
Make sense, thanks!

Reply:
pushed a commit